script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -159,7 +161,7 @@ But that glacier formed in an instant.
 [sha:a756d8ecb94b486da90c3b1e0f83c70b75e1d5e1]{
 -- Page 526, Offset 10011.
 -- 範囲は狭く、寒波は地を這うものだが、放たれてしまえば逃げる間もなく殺される。
-Its range is narrow, and the cold wave only crawls on the ground, but once it's released there would be no chance of escaping it.
+Its range is narrow, and the cold wave only crawls along the ground, but once it's released there would be no hope of escaping it.
 }
 [sha:41ed9c90a22fca2dd8fadcf78ba38586f400d38f]{
 -- Page 527, Offset 10012.
```

Comment:
10020:
"She's standing in that frigid air without a scratch on her...!"

I didn't even think algid was a real word (and neither does Firefox's spellcheck)

Reply:
Flowers for algid gone